### ISSUE TITLE
Ky create not working with typescript

### ISSUE THREAD

**Issue by alanhr** [sindresorhus/ky#256]
Hello , guys.
I'm face a problem when I generate the type definition using `ky.create({})`
The error is `Default export of the module has or is using private name 'TimeoutError'.` and `Default export of the module has or is using private name 'HTTPError'`

**Comment by sholladay** [comment]
Interesting. I guess we could use `export declare HTTPError`, etc. However, might it be a bit misleading given that we don't have a named export for `HTTPError` (we use `ky.HTTPError` instead)? Anyway, that's probably the way to go.

**Comment by alanhr** [comment]
@sholladay , when I added `export` in `TimeoutError` and `HTTPError` inside of `index.d.ts`  with  `declaration: true` everything worked  hell.

The output was :
```
declare const _default: {
    (url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    get(url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    post(url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    put(url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    delete(url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    patch(url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    head(url: import("ky").Input, options?: import("ky").Options | undefined): import("ky").ResponsePromise;
    create(defaultOptions: import("ky").Options): any;
    extend(defaultOptions: import("ky").Options): any;
    readonly stop: unique symbol;
    readonly TimeoutError: typeof import("ky").TimeoutError;
    readonly HTTPError: typeof import("ky").HTTPError;
};
```

**Comment by ulken** [comment]
My workaround for this is to type the assignment as `typeof ky`, i.e.

```ts
const client: typeof ky = ky.create({...});
```

But doesn't feel super-clean to have to do.

**Comment by sholladay** [comment]
Is this still happening with the latest version of Ky? If so, could you post a reproducible example for me to look at?

**Comment by ulken** [comment]
> Is this still happening with the latest version of Ky? If so, could you post a reproducible example for me to look at?

I'll check later tonight.

**Comment by ulken** [comment]
Installed `0.26`. Still happens to me.

**client.ts**
```ts
import ky from "ky":
const client = ky.create({});

Exported variable 'client' has or is using name 'TimeoutError' from external module "/Users/ulken/dev/aqua-iot-cloud-frontend/node_modules/ky/index" but cannot be named. ts(4023)
Exported variable 'client' has or is using name 'HTTPError' from external module "/Users/ulken/dev/aqua-iot-cloud-frontend/node_modules/ky/index" but cannot be named. ts(4023)
```

**Comment by mgcrea** [comment]
I think this is probably caused by the removing of module exports in #205, we should revert it, see #320.

**Comment by sindresorhus** [comment]
This should be fixed in v0.28.0: https://github.com/sindresorhus/ky/releases/tag/v0.28.0
